README.md

```diff
@@ -68,13 +68,22 @@ final class AppDelegate: ApplicationDelegate {
 		sudo apt update
 		sudo apt install vulkan-sdk -y
 		```
-	- Alternativelly you can try to install Vulkan SDK from tarballs provided by LunarG or if you are using Debian 12 and newer or Ubuntu 22.04 and newer you can substitute Vulkan SDK from LunarG by installing 	libvulkan-dev
+	- If you are using Debian 12+ or Ubuntu 22.04 and newer you can substitute Vulkan SDK from LunarG by installing libvulkan-dev
 		```
 		sudo apt install libvulkan-dev
 		```
-		But you will have to manually install `glslc` compiler from Gooogle's shaderc project located [here](https://github.com/google/shaderc).
+		If this doesn't work, you can try to install Vulkan SDK from tarballs provided by LunarG
+
+	- `glslc` compiler from Gooogle's shaderc project located [here](https://github.com/google/shaderc).   
+		There are packages for it available either in the above 'lunarg-vulkan-jammy' repo above, and/or with 23.04 and the 'universe' repo.. so use one of these or the other, but not both.
```

Comment:
This point makes no sense to be standalone like this. `glsls` is part of `vulkan-sdk` if it's installed from LunadG's reporsitory. If you don't use their repository this step should not suggest to use repo :)

Reply:
I understand the intent to improve readme and fully support it. What I'm suggesting is
- installation steps for Debian-based distributions with LunarG repository
- installation steps for newest Debian-based distributions without LunarG repository, but using whatever they provide
- completely manual installation from tarballs

Reply:
tried to separate it into lunarg-repo users, and everyone-else